packages/at_commons/lib/src/verb/sync_verb_builder.dart

```diff
@@ -9,6 +9,8 @@ class SyncVerbBuilder implements VerbBuilder {
 
   bool isPaginated = false;
 
+  int? skipDeletesUntil;
```

Comment:
Please take this opportunity to add `///` documentation for all of the parameters (commitId, regex, limit, isPaginated, skipDeletesUntil)

Reply:
added the dart docs